.github/workflows/build.yml

```diff
@@ -62,6 +62,10 @@ jobs:
         if: matrix.platform.rust_target == 'aarch64-apple-darwin'
         run: rustup target add aarch64-apple-darwin
 
+      - name: "Setup Rust"
+        if: matrix.platform.rust_target == 'x86_64-apple-darwin'
+        run: rustup target add x86_64-apple-darwin
```

Comment:
Are you sure this does something? Isn't that the default arch?

Reply:
Sorry for the late response, must have missed a notification somehwere. But I added this because I believe the default has changed for runners. At least in [this action output](https://github.com/L00Cyph3r/TrguiNG/actions/runs/10667475916) it says: `failed to build app: Target x86_64-apple-darwin is not installed (installed targets: aarch64-apple-darwin).`. So just to be sure we have build-targets available independent of which host-architecture we run on, I added the second action.

Reply:
Makes sense, rust probably changed which arch supports gets installed by default on arm macs.
Thanks.